packages/ui/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,33 @@
+import type { HTMLAttributes } from 'react';
+import TooltipTrigger from './Trigger';
+import TooltipContent from './Content';
+import { useTooltip } from './useTooltip';
+import { TooltipContext } from './TooltipContext';
+import * as S from './style.css';
+
+interface TooltipRootProps extends HTMLAttributes<HTMLDivElement> {
+  isOpen?: boolean;
+  defaultOpen?: boolean;
+}
+
+function TooltipRoot({ isOpen: open, defaultOpen = false, children, ...props }: TooltipRootProps) {
+  const { isOpen: internalOpen, triggerRef, contentRef, position } = useTooltip({ defaultOpen });
+
+  const isOpen = open ?? internalOpen;
+
+  return (
+    <TooltipContext.Provider value={{ isOpen, position, contentRef }}>
+      <div className={S.tooltipWrapper} ref={triggerRef} {...props}>
```

Comment:
clsx와 같은 유틸을 써서 안전하게 className을 결합해줘야 충돌이 안나요. 얘가 없다면 인스톨해주세요. 적용 안된 나머지 것들이 있다면 적용 모두 해주세요

```suggestion
      <div className={clsx(S.tooltipWrapper, props.className)} ref={triggerRef} {...props}>
```